src/mimefactory.rs

```diff
@@ -458,32 +458,45 @@ impl<'a> MimeFactory<'a> {
             self.from_addr.clone(),
         );
 
+        let undisclosed_recipients = match &self.loaded {
+            Loaded::Message { chat } => chat.typ == Chattype::Broadcast,
+            Loaded::Mdn { .. } => false,
+        };
+
         let mut to = Vec::new();
-        let email_to_remove = if self.msg.param.get_cmd() == SystemMessage::MemberRemovedFromGroup {
-            self.msg.param.get(Param::Arg)
+        if undisclosed_recipients {
+            to.push(Address::new_group(
+                "hidden-recipients".to_string(),
+                Vec::new(),
+            ));
```

Comment:
In case someone else reviews: I was a bit confused on first sight because link2xt changed `hidden-recipients: ;` to `hidden-recipients` here, but this works because he wraps it in `Address::new_group(..., Vec::new())`, which means exactly the same as `hidden-receipients: ;`: A group of email addresses called `hidden-recipients` that contains 0 addresses.

Reply:
This, used an API to create a new group instead of injecting group syntax manually.